nsv13/code/modules/overmap/traders.dm

```diff
@@ -169,10 +169,10 @@
 	desc = "Ship construction deeds done cheap (for a price)"
 	shortname = "CZC"
 	faction_type = FACTION_ID_NT
-	greetings = list("Welcome to CzanekCorp, we take cash, credit, and charge. What’cha need?",\
+	greetings = list("Welcome to CzanekCorp, we take cash, credit, and charge. What'cha need?",\
 	"CzanekCorp here. We got a new shipment in, you down for talking turkey?",\
 	"CzanekCorp, we got repairs and goods on a budget, you in?")
-	on_purchase = list("Yes, we know the tazers aren’t the safest, but if you don’t like ‘em, stop buying ‘em, eh?", "Good doing business with you. Good luck out there, killer.", "About time we got somebody who knows what they’re doing. Here, free shipping!", "No refunds, no returns!")
+	on_purchase = list("Yes, we know the tazers aren't the safest, but if you don't like 'em, stop buying 'em, eh?", "Good doing business with you. Good luck out there, killer.", "About time we got somebody who knows what they're doing. Here, free shipping!", "No refunds, no returns!")
```

Comment:
"Tazer"

I'll properly review this later, sorry.

Reply:
They're called that everywhere in the code, we talked about this before and if you want to change it I suggest doing it everywhere at once another time.